content/en/docs/ambient/install/platform-prerequisites/index.md

```diff
@@ -228,6 +228,11 @@ For example:
 
 OpenShift requires that `ztunnel` and `istio-cni` components are installed in the `kube-system` namespace, and that you set `global.platform=openshift` for all charts.
 
+Note: If your pod manifests include liveness or readiness probes, the OVN-Kubernetes CNI must be configured to run in `local` gateway mode for these probes to work
+properly. In this mode, traffic is routed through the host and applied to the host’s routing table. By default, OVN-Kubernetes runs in `shared` gateway mode, where
+traffic bypasses the host and OVS sends it directly to the node IP interface. To switch from `shared` to `local` gateway mode at runtime, follow the steps
+described [here](https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/ovn-kubernetes_network_plugin/configuring-gateway-mode#configuring-gateway-mode).
```

Comment:
```suggestion
Additionally, if your pod manifests include liveness or readiness probes, the OVN-Kubernetes CNI must be configured to run in `local` gateway mode for these probes to work
properly. In this mode, traffic is routed through the host and applied to the host’s routing table. By default, OVN-Kubernetes runs in `shared` gateway mode, where
traffic bypasses the host and OVS sends it directly to the node IP interface. To switch from `shared` to `local` gateway mode at runtime, follow the steps
described [here](https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/ovn-kubernetes_network_plugin/configuring-gateway-mode#configuring-gateway-mode).
```

Reply:
I rephrased it slightly to convey that its a one time config change as mentioned in the other PR - https://github.com/istio/istio.io/pull/16896#issuecomment-3348916284